### DISCUSSION TITLE
Monthly ESMValTool community meeting - April 2024

### DISCUSSION THREAD

**Discussion by paultgriffiths** [ESMValGroup/Community#161] [Meetings]
The next online community meeting will be on April 25th at 1300 BST, after EGU

The duration will be one hour.  [Join the meeting](https://ncas.zoom.us/j/93911612772?pwd=UmlaaTRXd1B3RHhHV3pQQjhnOHl2QT09)

Following @rswamina advice - @ESMValGroup/esmvaltool-developmentteam @valeriupredoi 

Calendar invitations to the meeting are sent to the mailing list, join it [here.](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list)

Format and agenda

Anyone with an interest in ESMValTool is welcome to join and ask questions

We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.

Using the input from the introduction round, we choose some topics to talk about in more detail.

If you already know that you would like to bring up a particular topic in the meeting, you can mention the GitHub issue on the topic in a comment below. That will make it easy to find for the other people in the meeting and we can already start discussing there before the meeting. Just create a new GitHub issue about your topic if there is no open issue yet.

GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the previous meeting are available in https://github.com/ESMValGroup/Community/discussions/155

**Comment by glpotter** [comment]
@paultgriffiths I probably can't make it for the meeting Thursday, I don't have much and I am waiting on news from Carlo Buontempo at ECMWF about Cmorizing reanalysis.

**Comment by paultgriffiths** [comment]
Updates
@rswamina - will work on tutorial for AI4PEX and OptimESM.
@malininae working on machine generated recipes, anyone using Cylc?
@ehogan UKMO uses Cylc a lot
@ehogan UKMO prepping hard for ESMValCore release.  Please look at the ESMValCore 2.11.0 milestones (https://github.com/ESMValGroup/ESMValCore/issues?q=is%3Aopen+is%3Aissue+milestone%3Av2.11.0) and merge ASAP or remove and add to the 2.12.0 milestone.
@paultgriffiths no meeting in May
